node/node.go

```diff
@@ -168,6 +168,16 @@ func New(conf *Config) (*Node, error) {
 	node.server.Config.DataDir = node.config.DataDir
 	// End Quorum
 
+	/* Quorum
+	// Check HTTP/WS prefixes are valid.
+	if err := validatePrefix("HTTP", conf.HTTPPathPrefix); err != nil {
```

Comment:
why are we removing this validation?

Reply:
it's a difference we have from the geth base.